content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -941,21 +1027,50 @@ the binary `/home/jane/bin/example-client-go-exec-plugin` is executed.
     exec:
       # Path relative to the directory of the kubeconfig
       command: "./bin/example-client-go-exec-plugin"
-      apiVersion: "client.authentication.k8s.io/v1beta1"
+      apiVersion: "client.authentication.k8s.io/v1"
+      interactiveMode: IfAvailable
 ```
 
 ### Input and output formats
 
 The executed command prints an `ExecCredential` object to `stdout`. `k8s.io/client-go`
 authenticates against the Kubernetes API using the returned credentials in the `status`.
-
-When run from an interactive session, `stdin` is exposed directly to the plugin. Plugins should use a
-[TTY check](https://godoc.org/golang.org/x/crypto/ssh/terminal#IsTerminal) to determine if it's
-appropriate to prompt a user interactively.
+The executed command is passed an `ExecCredential` object as input via the `KUBERNETES_EXEC_INFO`
+environment variable. This input contains helpful information like the expected API version
+of the returned `ExecCredential` object and whether or not the plugin can use `stdin` to interact
+with the user.
+
+When run from an interactive session (i.e., a terminal), `stdin` can be exposed directly
+to the plugin. Plugins should use the `spec.interactive` field of the input
+`ExecCredential` object from the `KUBERNETES_EXEC_INFO` environment variable in order to
+determine if `stdin` has been provided. A plugin's `stdin` requirements (i.e., whether
+`stdin` is optional, strictly required, or never used in order for the plugin
+to run successfully) must be declared via the `user.exec.interactiveMode` field in the
+[kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) (see table
+below for valid values).
+
+| `interactiveMode` Value | Meaning |
```

Comment:
@PI-Victor do you know of a way that I can ensure that this table renders without splitting words on different lines?
![image](https://user-images.githubusercontent.com/3081590/122459911-7ecb9700-cf7f-11eb-9d4d-512dff114e03.png)

Reply:
sorry i missed this, @sftim have any idea on this one? does it need some special css code to fix this?

Reply:
Ah, HTML table layout.

I would raise a separate issue about the rendering and label it area/web-development.

Reply:
raised issue here: https://github.com/kubernetes/website/issues/28815

unfortunately couldn't coax the bot into applying the area/web-development...perhaps someone can enlighten me